counting_sort/counting_sort.py

```diff
@@ -0,0 +1,38 @@
+#Running time Complexity: O(z) where z = max(len(a), max(a))
+#Auxillary Space: O(n)
+def count_occurrences(a, c):
+    """
+    :param a: Iterable of elements
+    :param c: Intermediate variable
+    :return: No returns, counts occurrences
+    Time Complexity: O(n)
+    """
+    for i in a:
+        c[a[i]] += 1
+    for i in range(1, len(c)):
+        c[i] = c[i - 1] + c[i]
```

Comment:
@aashutoshrathi  Hey! I have incorporated your suggestions. But, it failed again. Please help me out.